sdcm/cluster_k8s/__init__.py

```diff
@@ -2208,12 +2208,12 @@ def fstrim_scylla_disks(self):
             scylla_disk_path = "/mnt/raid-disks/disk0/"
             namespace = "default"
         else:
-            pods_selector = "app.kubernetes.io/instance=local-csi-driver"
+            pods_selector = "app.kubernetes.io/name=local-csi-driver"
             scylla_disk_path = "/mnt/persistent-volumes"
             namespace = "local-csi-driver"
         podnames = self.parent_cluster.k8s_cluster.kubectl(
             f"get pod -l {pods_selector} --field-selector spec.nodeName={self.node_name} "
-            "-o jsonpath='{range .items[*]}{.metadata.name}'",
+            """-o jsonpath='{range .items[*]}{.metadata.name}{"\\n"}{end}'""",
```

Comment:
The referenced CI job failed with the following error:
```
19:34:46  error: error parsing jsonpath {range .items[*]}{.metadata.name}{"
19:34:46  "}{end}, unterminated quoted string
```

The error mentioned in the commit message `error: pod, type/name or --filename must be specified` is caused by not found pod. So, fix of the selector must be sufficient.